src/extension.js

```diff
@@ -11,6 +11,10 @@ const constants = require('./constants');
 const _ = require('lodash');
 const getPackageDeepFiles = require('./get-package-deep-files');
 
+function isRequire(line) {
```

Comment:
Should this be pulled into it's own file to prevent duplication between here and get-position?